extensions/amp-ad/0.1/amp-ad-xorigin-iframe-handler.js

```diff
@@ -94,6 +95,19 @@ export class AmpAdXOriginIframeHandler {
         .then(info => {
           this.element_.creativeId = info.data.id;
         });
+    listenFor(this.iframe, 'get-html', (info, source, origin) => {
+      if (!this.iframe) {
+        return;
+      }
+
+      const {selector, attributes} = info;
+      const content = getHtml(window, selector, attributes);
```

Comment:
@cramforce I'm not sure I did it correctly. I have not found anywhere else in the code `this.win`, so I created it.